src/main.py

```diff
@@ -181,6 +185,12 @@ def handler(event, context):
 
         jsonschema.validate(event, schema)
 
+        log_level = event.get('log_level')
+        if log_level:
+            level = logging.getLevelName(log_level)
```

Comment:
Missing `import logging`

Reply:
Oops, I've just forgotten it...

Fixed @ https://github.com/cohalz/CertUpdater/pull/6/commits/121b7d892b5305750650f74e44cec4b4f8a04278